packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -551,33 +578,68 @@ function ResizableTableColumnHeader(props) {
     ];
     return options;
   }, [allowsSorting]);
-  // if we're resizing another column, then hover shouldn't show the resizer of a different column
-  let showResizer = (isHovered && !columnState.currentlyResizingColumn) || columnState.currentlyResizingColumn === column.key;
 
   useEffect(() => {
     if (columnState.currentlyResizingColumn === column.key) {
-      focusSafely(ref.current);
+      // focusSafely won't actually focus because the focus moves from the menuitem to the body during the after transition wait
+      resizingRef.current.focus();
     }
   }, [columnState.currentlyResizingColumn, column.key]);
 
+  let showResizer = headerRowHovered || columnState.currentlyResizingColumn != null;
```

Comment:
I noticed that this displays all the resizers when a column is being resized, is that what we want? Is it mainly for the benefit of mobile users so they know that other columns are resizable?

Reply:
that's correct